### ISSUE TITLE
Keyboard input is repeated unexpectedly on remote Linux

### ISSUE THREAD

**Issue by rustdesk** [rustdesk/rustdesk#189]
RT

**Comment by zclgudu1224** [comment]
Hi, I found some problems during the use. The MAC remote ubantu 20.04 found that occasionally, pressing a child parent will keep brushing the letter, and pressing the delete key is the same. Obviously, I haven't clicked the keyboard or kept deleting. I'm not sure if there is an occasional network fluctuation problem, but the network fluctuation should not occur. I hope it can be solved

**Comment by rustdesk** [comment]
> solved
Yes, related to network fluctuation, solved in https://github.com/rustdesk/rustdesk/issues/189

**Comment by zclgudu1224** [comment]
Ths!

**Comment by open-trade** [comment]
try out https://github.com/rustdesk/rustdesk/releases/download/1.1.8/rustdesk-1.1.8.deb
